oci-library/oci-hol/java-management/set-up-logrotate/set-up-logrotate.md

```diff
@@ -0,0 +1,138 @@
+# Set up logrotate
+
+## Introduction
+
+This lab will guide you in setting up the logrotate utility on your host machine to periodically rotate the `usagetracker.log` generated by Java Usage Tracker.
+
+Although the size of the Java Usage Tracker log file is small, you should consider periodically truncate, compress, archive, or delete the log file. This is because the Java Usage Tracker does not check for available disk space or perform these administrative tasks on the log file as it incrementally adds records to it. 
+
+Estimated Time: 10 minutes
+
+### Objectives
+
+In this lab, you will:
+
+* Set up logrotate on Linux instances
+* Set up logrotatewin on Windows instances
+
+### Prerequisites
+
+* Access to your host machine with management agent already setup
+* Completed labs 1 to 4
+
+## Task 1: Set up logrotate on Linux instances
+
+1. Run the following two commands:
+
+    ```
+    <copy>
+    sudo yum install logrotate
+    sudo touch /etc/logrotate.d/jms
+    </copy>
+    ```
+
+2. Open the file /etc/logrotate.d/jms in a text editor: 
```

Comment:
Can consider rephrasing to "`In the Terminal window, create a logrotate.d/jms file by entering this command: ...`" to standardize the instructions for file creation. Reference: https://apexapps.oracle.com/pls/apex/dbpm/r/livelabs/workshop-attendee-2?p210_workshop_id=912&p210_type=2&session=9626690155642 (Lab 2, Task 3, Step 3)